node/src/accountant/mod.rs

```diff
@@ -3381,429 +4099,450 @@ mod tests {
+        let finish_scan_params_arc = Arc::new(Mutex::new(vec![]));
+        let payable_notify_later_params_arc = Arc::new(Mutex::new(vec![]));
+        let system = System::new(test_name);
+        let mut subject = AccountantBuilder::default()
+            .logger(Logger::new(test_name))
             .build();
-        let service_rate = i64::MAX as u64;
+        subject
+            .scanners
+            .replace_scanner(ScannerReplacement::Payable(ReplacementType::Mock(
+                ScannerMock::default()
+                    .finish_scan_params(&finish_scan_params_arc)
+                    .finish_scan_result(PayableScanResult {
+                        ui_response_opt: None,
+                        result: OperationOutcome::Failure,
+                    }),
+            )));
+        // Important. Otherwise, the scan would've been handled through a different endpoint and
+        // gone for a very long time
+        subject
+            .scan_schedulers
+            .payable
+            .inner
+            .lock()
+            .unwrap()
+            .last_new_payable_scan_timestamp = SystemTime::now();
+        subject.scan_schedulers.payable.new_payable_notify_later = Box::new(
+            NotifyLaterHandleMock::default().notify_later_params(&payable_notify_later_params_arc),
+        );
+        let sent_payable = SentPayables {
+            payment_procedure_result: Err(PayableTransactionError::Sending {
+                msg: "booga".to_string(),
+                hashes: vec![make_tx_hash(456)],
+            }),
+            response_skeleton_opt: None,
+        };
+        let addr = subject.start();
 
-        subject.record_service_consumed(service_rate, 1, SystemTime::now(), 2, &wallet);
+        addr.try_send(sent_payable.clone())
+            .expect("unexpected actix error");
 
-        TestLogHandler::new().exists_log_containing(&format!(
-            "ERROR: Accountant: Overflow error recording consumed services from {}: total charge {}, service rate {}, byte rate 1, payload size 2. Skipping",
-            wallet,
-            i64::MAX as u64 + 1 * 2,
-            i64::MAX as u64
-        ));
+        System::current().stop();
+        assert_eq!(system.run(), 0);
+        let mut finish_scan_params = finish_scan_params_arc.lock().unwrap();
+        let (actual_sent_payable, logger) = finish_scan_params.remove(0);
+        assert_eq!(actual_sent_payable, sent_payable,);
+        test_use_of_the_same_logger(&logger, test_name, None);
+        let mut payable_notify_later_params = payable_notify_later_params_arc.lock().unwrap();
+        let (scheduled_msg, _interval) = payable_notify_later_params.remove(0);
+        assert_eq!(scheduled_msg, ScanForNewPayables::default());
+        assert!(
+            payable_notify_later_params.is_empty(),
+            "Should be empty but {:?}",
+            payable_notify_later_params
+        );
```

Comment:
You'd like to configure the notify later of the pending payable and assert on it to verify that it stays empty.